lib/components/user/user-account-screen.js

```diff
@@ -69,6 +70,11 @@ class UserAccountScreen extends Component {
     this.props.routeTo('/')
   }
 
+  _handleSendPhoneVerificationCode = newPhoneNumber => {
```

Comment:
is this method name a typo? (since `_handleRequestPhoneVerificationCode` is mentioned but not defined in the diff)

Reply:
Yep this was a typo, updated in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/337d3b0a77ffbf0322683910f3e1647f79dc4d6b